src/schema/v2/partner/partner.ts

```diff
@@ -853,6 +853,28 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           return Array.from(new Set(cities)).filter(Boolean)
         },
       },
+      contact: {
+        type: ContactType,
+        description: "A Singular Contact belonging to the Partner",
+        args: {
+          contactId: {
+            type: new GraphQLNonNull(GraphQLString),
+            description: "The slug or ID of the Contact",
+          },
+        },
+        resolve: async ({ id }, args, { partnerContactLoader }) => {
+          const { contactId } = args
+
+          if (!partnerContactLoader) return null
+
+          const contact = await partnerContactLoader({
+            partnerId: id,
+            contactId: contactId,
+          })
+
+          return contact ?? null
```

Comment:
Maybe just return contact? No reason to handle the ?? case as the system should do this for you

Reply:
You got it